docs/web/docs/1-getting_started/7-github.mdx

```diff
@@ -191,6 +191,28 @@ By installing IAMbic GitHub App, the app can integrate the pull-request and merg
 12. Click "Generate a private key".
 13. The private key should be downloaded automatically.
 
+
+## Create an Additional GitHub Repository for IAMbic Plan and Apply Summaries
+
+The IAMbic GitHub Application generates Plan and Apply Summaries for each pull request in your IAMbic templates repository.
+These summaries provide an overview of the changes proposed by a pull request, and the results of applying those changes.
+The IAMbic Github Application stores these summaries in a separate repository to allow for easier tracking and management,
+and as it adds comments in each pull request, it provides a link to the summary in this separate repository.
+
+
+1. **Create a new GitHub repository** in your organization that will host the IAMbic Plan and Apply Summaries.
+   The repository must have the same name as your IAMbic templates repository, with `-gist` appended to the end.
+   For example, if your repoistory name is `iambic-templates`, the new repository must be named `iambic-templates-gist`.
+
+2. **Ensure the IAMbic GitHub App has write access to the new repository.
+   - To set the repository permissions:
+     1. Go to your organization settings (https://github.com/organizations/ExampleOrg/settings/profile).
+     2. Select "GitHub Apps" under "Third-party Access" in the left navigation bar.
+     3. Click "Configure" on "Iambic Integrations".
```

Comment:
you might mention that if a user has accounts in 2 separate partitions in AWS, they may need 2 separate Github Apps. 1 for Each Partition.